midterm-2023-11-07/code/5041-filip-zhikovski/scripts.ts

```diff
@@ -0,0 +1,202 @@
+interface Genre {
+    genre:string;
+}
+interface Cast {    
+    actor: string,
+    character: string
+}
+interface Oscar {
+    award: string
+}
+
+
+
+interface Movie{
```

Comment:
Mostly correct interfaces +8 - while `genre` is an array, it's an array of strings. And `oscars` is not an array at all, it's an object